.github/workflows/q-update-issue-scheduler.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: quarterly-update-issues
+on:
+  schedule:
+    - cron: "21 8 2 2,5,8,11 *" # 8:21 AM UTC, second of feb, may, aug, nov
```

Comment:
I would add the `workflow_dispatch:` option too so we can test it out once this is merged into `main`.

Reply:
what does that entail?

Reply:
Nope that's it!